### PR TITLE
Add support for 'payment-flow' query param for the web-sdk bridge

### PR DESCRIPTION
**Pull request by gregjopa**
This PR adds support for the new `payment-flow` query parameter for the web-sdk bridge url.

### CODE REVIEW

**Review by wsbrunson — APPROVED** [review]

File: server/meta.jsx [hunk 1]
```diff
@@ -91,17 +91,31 @@ function validateWebSDKUrl({ pathname, query }) {
     );
   }
   // check for extraneous parameters
-  Object.keys(query).forEach((param) => {
-    if (param !== "version" && param !== "origin") {
+  const validWebSDKBridgeParams = ["origin", "version", "payment-flow"];
```
Comment (dtjones404) [inline server/meta.jsx:94]: should we add 'debug' here?

Comment (gregjopa) [reply server/meta.jsx:94]: Good point @dtjones404! I'll send a follow up PR for adding support for the optional `debug` param.

Comment (gregjopa) [reply server/meta.jsx:94]: Good point @dtjones404! I'll send a follow up PR for adding support for the optional `debug` param.

Comment (gregjopa) [reply server/meta.jsx:94]: @dtjones404 here's the follow up PR to add support for the `debug` query param: https://github.com/paypal/paypal-sdk-client/pull/219

**Pull request merged**